recipes/sisl/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "sisl" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "29f995077180dc43d30b6c55a2e933182912661d8ed8cfaf3f4c4ac40306fe91" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/zerothi/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [not (py27 or py>34)]
+
+requirements:
+  build:
+    # We need the toolchain for gfortran compiler as well as libgfortran/gcc
+    - toolchain
+    - libgfortran  # [linux]
+    - gcc
+    - python
+    - six  # [py27]
+    - numpy 1.9.*
+    - netcdf4
+    - scipy
```

Comment:
Are `scipy` and `netcdf4` actually needed at build time? If they're just `install_requires`, then `--single-version-externally-managed` means that you don't actually need them present at build time.

Reply:
You are most likely correct, never thought about it. will check.